pandas/tests/io/excel/test_writers.py

```diff
@@ -1115,6 +1115,32 @@ def test_bytes_io(self, engine):
             reread_df = pd.read_excel(bio, index_col=0)
             tm.assert_frame_equal(df, reread_df)
 
+    def test_engine_kwargs(self, engine, path):
+        df = DataFrame([{"A": 1, "B": 2}, {"A": 3, "B": 4}])
```

Comment:
Can you add `# GH#52368` as the first line

Reply:
Should be fixed on the latest push